src/GitTools.Testing/Fixtures/RepositoryFixtureBase.cs

```diff
@@ -18,7 +18,7 @@ protected RepositoryFixtureBase(Func<string, IRepository> repoBuilder)
 
         protected RepositoryFixtureBase(IRepository repository)
         {
-            _sequenceDiagram = new SequenceDiagram();
+            this._sequenceDiagram = new SequenceDiagram();
```

Comment:
+1 to the use of `this.`, but can you please have the underscore removed as well?

```suggestion
            this.sequenceDiagram = new SequenceDiagram();
```

Reply:
I will remove them manually but afaik the analyzers don't have a rule to catch this case (nor a code-fix to edit it).